lib/rex/exploitation/cmdstager/jsp.rb

```diff
@@ -0,0 +1,114 @@
+# -*- coding: binary -*-
+
+class Rex::Exploitation::CmdStagerJSP < Rex::Exploitation::CmdStagerBase
+  JSP_KEYWORDS = %w[abstract assert boolean break byte case catch char class const continue default
+                     do double else enum extends final finally float for goto if implements import
+                     instanceof int interface long native new package private protected public return
+                     short static strictfp super switch synchronized this throw throws transient try
+                     void volatile while true false null var _
```

Comment:
Why the `_` ? 👀 

Seems like it should be valid to me: https://docs.oracle.com/cd/E13222_01/wls/docs90/webapp/reference.html

Reply:
I think that's a Java restriction: https://www.geeksforgeeks.org/using-_-underscore-variable-name-java/

Reply:
Also, I think we should also add any namespace or class name accessible from the scope of the variables we are generating, for example, System is not a Java or JSP keyword, but we can't have it as a variable name. this isn't something the `Rex::RandomIdentifier::Generator` should be aware of, I think each module can have its own list (one exploit might import `java.util.ArrayList`, `ArrayList` would become a reserved word variables can't use).

Reply:
Good call :+1: